docs/user-guide.rst

```diff
@@ -166,7 +166,7 @@ Request Data
 Headers
 ~~~~~~~
 
-You can specify headers as a dictionary in the ``headers`` argument in :meth:`~poolmanager.PoolManager.request`:
+You can specify headers as a dictionary in the ``headers`` argument in :meth:`~urllib3.request`:
```

Comment:
`urllib3.request` is a `func`, not a `meth`. This applies to all the changes below.

It's also not in the reference. To do that, you could add a `docs/reference/urllib3.rst` file:

```
urllib3 top level
=================

.. autofunction:: urllib3.request
```

and add `urllib3` in `reference/index.rst`.